games_services/android/src/main/kotlin/com/abedalkareem/games_services/Auth.kt

```diff
@@ -40,7 +40,7 @@ class Auth : PluginRegistry.ActivityResultListener {
     result: MethodChannel.Result
   ) {
     activity ?: return
-    val signInOption = GoogleSignInOptions.Builder(GoogleSignInOptions.DEFAULT_GAMES_SIGN_IN)
+    val signInOption = GoogleSignInOptions.Builder(GoogleSignInOptions.DEFAULT_GAMES_SIGN_IN).requestEmail()
```

Comment:
Maybe we should pass a flag like "shouldEnableSavedGame" because not everyone is interested in getting the user's email and it will show the user that the app is asking for your email. What do you think? can you add it?

Reply:
I decided to move away from this solution. I wanted this because I did not want to implement auth, but because of other things I must.
This PR was also out of the scope of my knowledge, but GPT walked me trough somehow... Considering these, I'am sorry but can not do that.